spynnaker/pyNN/overridden_pacman_functions/graph_edge_filter.py

```diff
@@ -75,6 +79,11 @@ def _add_edge_to_new_graph(
     @staticmethod
     def _is_filterable(edge, graph_mapper):
         app_edge = graph_mapper.get_application_edge(edge)
+        if hasattr(app_edge, "synapse_information"):
```

Comment:
check that its a instance of projection_application_edge instead of for a attr, what about a future impl that has that property but isnt what you expected it to be? duck typing is dangerious